docs/man_pages/project/configuration/install.md

```diff
@@ -0,0 +1,23 @@
+install
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns install [--path]`
+
+Installs all platforms and dependencies described in package.json file.
```

Comment:
Apply this comment to the descriptions of the command that appear in index.md and the affected related commands.